R/outputs.R

```diff
@@ -672,18 +672,28 @@ save_output <- function(filename, dir,
 generate_output_summary_report <- function(report_path,
                                            output_zip,
                                            quiet = FALSE) {
-  rmd_path <- system_file("report/summary_report.Rmd")
-
-  report_path_dir <- normalizePath(dirname(report_path), mustWork = TRUE)
   report_filename <- basename(report_path)
+  report_path_dir <- normalizePath(dirname(report_path), mustWork = TRUE)
   output_zip_path <- normalizePath(output_zip, mustWork = TRUE)
-  rmarkdown::render(rmd_path, params = list(
+  ## Render uses relative paths to locate the html file and pacakge author
+  ## advises against using output_dir see:
+  ## https://github.com/rstudio/rmarkdown/issues/587#issuecomment-168437646
+  ## so set up a temp directory with all report sources and generate from there
+  tmpd <- tempfile()
```

Comment:
You might use withr here for these , but this is ok too

Reply:
`withr` is nicer, I have switched that over